dockerfiles/deps/base.sh

```diff
@@ -7,8 +7,8 @@ PANDOC_VERSION=${PANDOC_VERSION:-"3.1.9"}
 
 echo "GPU enabled: $GPU_ENABLED"
 
-dnf -y update
-dnf -y upgrade
+dnf -y update && dnf -y upgrade
+dnf -y install epel-release
```

Comment:
Are these changes for the rockylinux images?

Reply:
yes, wolfie doesn't rely on any of those 3 scripts